src/components/ClusterManager.js

```diff
@@ -310,7 +310,7 @@ export default class ClusterManager extends PureComponent {
         disabled: !canCompute,
         style: { marginRight: '2rem', ...styles.verticalCenter },
         ...(isRStudioImage ? {} : Utils.newTabLinkProps)
-      }, [isRStudioImage ? img({ src: rLogo, style: { maxWidth: 24, maxHeight: 24 } }) : icon('terminal', { size: 24 })]),
+      }, [isRStudioImage ? img({ src: rLogo, alt: 'R-Studio Logo', style: { maxWidth: 24, maxHeight: 24 } }) : icon('terminal', { size: 24 })]),
```

Comment:
Nit: I think it should be "RStudio" (without the dash). That's what they seem to do consistently at https://rstudio.com/. However, now that I think about it, maybe "RStudio" isn't handled well by screen readers?